docs/sources/flow/reference/components/loki.source.azure_event_hubs.md

```diff
@@ -0,0 +1,129 @@
+---
+title: loki.source.azure_event_hubs
+---
+
+# loki.source.azure_event_hubs
+
+`loki.source.azure_event_hubs` receives Azure Event Hubs messages by making use of an Apache Kafka
+endpoint on Event Hubs. For more information, see
+the [Azure Event Hubs documentation](https://learn.microsoft.com/en-us/azure/event-hubs/azure-event-hubs-kafka-overview).
+
+To learn more about streaming Azure logs to an Azure Event Hubs, you can see
+this [tutorial](https://learn.microsoft.com/en-us/azure/active-directory/reports-monitoring/tutorial-azure-monitor-stream-logs-to-event-hub).
+
+Note that an Apache Kafka endpoint is not available within the `Basic` pricing plan. For more information, see
+the [Event Hubs pricing page](https://azure.microsoft.com/en-us/pricing/details/event-hubs/).
+
+Multiple `loki.source.azure_event_hubs` components can be specified by giving them
+different labels.
+
+## Usage
+
+```river
+loki.source.azure_event_hubs "LABEL" {
+	fully_qualified_namespace = "HOST:PORT"
+	event_hubs                = EVENT_HUB_LIST
+	forward_to                = RECEIVER_LIST
+
+	authentication {
+		mechanism = "AUTHENTICATION_MECHANISM"
+	}
+}
+```
+
+## Arguments
+
+`loki.source.azure_event_hubs` supports the following arguments:
+
+ Name                        | Type                 | Description                                                                                                                                                             | Default                          | Required 
+-----------------------------|----------------------|-------------------------------------------------------------------------------------------------------------------------------------------------------------------------|----------------------------------|----------
+ `fully_qualified_namespace` | `string`             | Event hub namespace. Typically, it looks like <your-namespace>.servicebus.windows.net:9093                                                                              |                                  | yes      
```

Comment:
I'd prefer trying to keep this table as small as possible, and moving detailed descriptions or examples outside of the table:

```suggestion
 `fully_qualified_namespace` | `string`             | Event hub namespace.                                                                             |                                  | yes      
```